client/my-sites/exporter/exporter.jsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import CompactCard from 'components/card/compact';
+import Gridicon from 'components/gridicon';
+import Button from 'components/forms/form-button';
+import AdvancedOptions from 'my-sites/exporter/advanced-options';
+
+export default React.createClass( {
+	displayName: 'Exporter',
+
+	render: function() {
+		return (
+			<div className="section-export">
+				<CompactCard>
+					<header>
+						<Button
+							className="exporter__export-button"
+							disabled={ false }
+							isPrimary={ true }
+						>
+							{ this.translate( 'Export' ) }
+						</Button>
+						<h1 className="exporter__title">
+							{ this.translate( 'Download an Export File' ) }
+						</h1>
+					</header>
+					<a href="#" onClick={ this.props.toggleAdvancedSettings }>
+						<Gridicon
+							icon={ this.props.advancedSettings.isVisible ? 'chevron-up' : 'chevron-down' }
+							size={ 16 } />
+						{ this.translate( 'Advanced Export Settings' ) }
+					</a>
+				</CompactCard>
+
+				{
+					this.props.advancedSettings.isVisible &&
+					<AdvancedOptions
```

Comment:
This makes perfect sense in a case where the depth is 2 or 3, but with more complicated UIs to me it makes less and less sense,  both because more independent components emerge, and because we seem to be passing data that’s not obvious how we need.

I have an idea that, in my mind, looks more logical:

* We merge `Exporter` and `ExporterContainer`
* We extract `CompactCard` into `BasicSettings`

Reasoning:

* I seriously doubt we will reuse the exporter without the need of the data connection, but keeping them together save us a level of nesting and redirection (even if we need at some point, extraction is trivial).
* Now both of the settings components will only get the data they need and we won’t have the `Exporter`, which gets all of the settings only to pass them to a child.

P.S. Don’t treat this as a blocker, it’s more of a philosophical exploration for going forward.
P.P.S. Why do we have options and settings? Do they mean the same thing?